doc/assessment.md

```diff
@@ -77,35 +106,133 @@ An assessment is broken down in to a list of zones, like this:
 ],
 ```
 
-* Each zone appears in the given order in the assessment. Zone titles are optional and are displayed to the student if present.
+Zone Property | Type | Description
+--- | --- | ---
+`title` | string | The title of the zone. (Optional; default: none)
+`questions` | array | The list of slots for questions and question alternatives within the zone. (Optional; default: none)
+`numberChoose` | integer | Number of questions to select for each student from this zone. (Optional; default: select all)
+`maxPoints` | number | Limit on the number of points that can be earned from this zone. (Optional; default: sum of question max points)
+`bestQuestions` | integer | Only this many questions in the zone will count towards the total points (highest-point questions will count). (Optional; default: use all questions)
+
+## `questions`: Slots for questions and question alternatives
+
+Each zone has a list of *slots* given by the `questions` array. Each slot contains either a single question `id`:
+
+```json
+{"id": "hardQV1", "points": 10}
+```
+
+Or a slot can contain a *question alternative list*:
+
+```json
+{
+    "numberChoose": 1,
+    "points": 5,
+    "alternatives": [
+        {"id": "FirstAltQ"},
+        {"id": "SecondAltQ"}
+    ]
+}
+```
+
+Slot property | Type | Description
+--- | --- | ---
+`points` | number or array | The number of points for this question. Can be a number (e.g., `10`) or a declining list of points (e.g., `[10, 8, 4]`) for partial credit on Exams. (Required; no default)
+`maxPoints` | number | The maximum points available for this question on a Homework that allows multiple attempts for more points. (Optional: default: same as `points`)
+`id` | string | The question ID if this slot contains just one question (can’t be specified with `alternatives`). (Optional; default: none)
+`alternatives` | array | The list of question alternatives if this slot contains multiple alternative questions (can’t be specified with `id`). (Optional; default: none)
+`numberChoose` | integer | If `alternatives` are specified, the number of them to select. (Optional; default `1`).
+`triesPerVariant` | integer | The maximum number of attempts allowed for each question variant (on Homeworks). (Optional; default `1`)
+`forceMaxPoints` | boolean | Whether to force all students to receive maximum points. See [Regrading](regrading.md). (Optional; default `false`)
+
+## Question alternative lists
+
+
+
```

Comment:
Not sure why this code block is here? I think this is a `TODO:`?

Reply:
yep, this goes with the `FIXME` below it 😄